### PR TITLE
modified ProgramContainer component and removed <Button> on North County Page

### PR DESCRIPTION
**Pull request by jeremyjo**
- modified ProgramContainer component to allow <Button> to be removed if the `href` attribute is not present
- Removed `<Button>` from North County Page for all of the tracks on this page

### CODE REVIEW

File: src/components/ProgramContainer.js [hunk 1]
```diff
@@ -33,12 +33,20 @@ const ProgramContainer = ({
 
 				{programFootText && <p className='paragraph'>{programFootText}</p>}
 
-				<a href={href} target='_blank' className='btn-wrapper'>
+				{/* Modified the <a> and <Button> allow the <Button> to be removed if there is no href attribute */}
```
Comment (DLoftis9) [inline src/components/ProgramContainer.js:36]: you don't need this comment it can be removed

File: src/components/ProgramContainer.js [hunk 2]
```diff
@@ -33,12 +33,20 @@ const ProgramContainer = ({
...
+				{href && <a href={href} target='_blank' className='btn-wrapper'>
 					<Button
 						children={`Click to sign up!`.toUpperCase()}
 						additionalClass={btnColor}
 					/>
-				</a>
+				</a>}
+
+				{/* <a href={href} target='_blank' className='btn-wrapper'>
```
Comment (DLoftis9) [inline src/components/ProgramContainer.js:44]: this commented code can be removed

File: src/pages/programs/north-county-and-expansion.js [hunk 3]
```diff
@@ -187,7 +187,8 @@ const NorthCountyAndExpansion = ({ componentName }) => {
 					programDescription={`Students will learn music fundamentals and music production skills including navigating a Digital Audio Workstation (DAW), sampling, creating percussive loops, melody composition, and mixing.`}
 					programFootText='Students will then use their music technology knowledge and skills to create their own song! Some of the technologies that will be covered will be: SoundTrap, Novation LaunchPad, Playground Sessions and more.'
 					btnColor='purple'
-					href='https://www.hisawyer.com/treobytes/schedules/activity-set/673279'
+					// intending to remove the buttons on this page
```
Comment (DLoftis9) [inline src/pages/programs/north-county-and-expansion.js:190]: these commented lines (190-191, 210-211, 230-231, 250-251) can be removed. It's explanatory in the ProgramContainer component

**Comment by jeremyjo** [comment]
I pushed a new version of dev-CT3 to remove these comments. I don't see a way to do a new pull request. Would the current pull request reflect today's version of dev-CT3?

**Review by DLoftis9 — APPROVED** [review]

**Review by DLoftis9 — APPROVED** [review]

**Pull request merged**